pkgs/native_assets_cli/lib/src/validation.dart

```diff
@@ -8,72 +8,77 @@ import '../native_assets_cli_builder.dart';
 
 typedef ValidationErrors = List<String>;
 
-Future<ValidationErrors> validateBuildConfig(BuildConfig config) async =>
-    _validateHookConfig(config);
+Future<ValidationErrors> validateBuildInput(BuildInput input) async =>
+    _validateHookInput(input);
 
-Future<ValidationErrors> validateLinkConfig(LinkConfig config) async {
+Future<ValidationErrors> validateLinkInput(LinkInput input) async {
   final errors = <String>[
-    ..._validateHookConfig(config),
+    ..._validateHookInput(input),
   ];
-  final recordUses = config.recordedUsagesFile;
+  final recordUses = input.recordedUsagesFile;
   if (recordUses != null && !File.fromUri(recordUses).existsSync()) {
-    errors.add('Config.recordUses ($recordUses) does not exist.');
+    errors.add('Input.recordUses ($recordUses) does not exist.');
   }
   return errors;
 }
 
-ValidationErrors _validateHookConfig(HookConfig config) {
+ValidationErrors _validateHookInput(HookInput input) {
```

Comment:
The other validation I saw earlier uses a name parameter, would it make sense to do that also here so error messages get  `"BuildInput.."` or `"LinkInput..."`)